cypress/integration/unit/util/buildParentXpath_spec.js

```diff
@@ -0,0 +1,17 @@
+import { buildParentXpath } from '../../../../src/util/buildParentXpath'
+
+describe('buildParentXpath', () => {
+  it('concatenates parent and child xpath', () => {
+    const parent = 'parent'
+    const child = 'child'
+
+    expect(buildParentXpath(parent, child)).to.eq('parent/child')
+  })
+
+  it('does not concatenate parent and child xpath if child is absolute', () => {
```

Comment:
Might seem minor, but assuming this catches `//` as well, can we add a test for that?